tools/build-rawtherapee

```diff
@@ -102,22 +139,19 @@ fi
 
 # Quit if no updates and build-only flag not set
 if [[ "$cloned" = "false" && "$buildOnly" = "false" && "$updates" = "false" && "$existsExe" = "true" ]]; then
-    printf '%s\n' "No updates, nothing to do."
+    printf '%s\n' "No ~, nothing to do."
```

Comment:
Hm, what's the intended meaning of `~` here?

Reply:
I don't know, looks like a result of a failed search&replace operation :sweat_smile: 
I removed the change in that line.